packages/url_launcher/url_launcher_web/example/README.md

```diff
@@ -1,24 +1,12 @@
 # Testing
 
-This package utilizes the `integration_test` package to run its tests in a web browser.
+This package uses `package:integration_test` to run its tests in a web browser.
 
-See [flutter.dev > Integration testing](https://flutter.dev/docs/testing/integration-tests) for more info.
+See [Plugin Tests > Web Tests](https://github.com/flutter/flutter/wiki/Plugin-Tests#web-tests)
+in the Flutter wiki for instructions to setup and run the tests in this package.
 
-## Running the tests
-
-Make sure you have updated to the latest Flutter master.
-
-1. Check what version of Chrome is running on the machine you're running tests on.
-
-2. Download and install driver for that version from here:
-    * <https://chromedriver.chromium.org/downloads>
-
-3. Start the driver using `chromedriver --port=4444`
-
-4. Run tests: `flutter drive -d web-server --browser-name=chrome --driver=test_driver/integration_test_driver.dart --target=integration_test/TEST_NAME.dart`, or (in Linux):
-
-    * Single: `./run_test.sh integration_test/TEST_NAME.dart`
-    * All: `./run_test.sh`
+Check [flutter.dev > Integration testing](https://flutter.dev/docs/testing/integration-tests)
+for more info.
 
 ## Mocks
```

Comment:
Mocks are also included in the Wiki above. Let's replace the whole  contents of this section by:

```
See [Plugin Tests > Web Tests > Mocks](https://github.com/flutter/flutter/wiki/Plugin-Tests#mocks)
in the Flutter wiki for more information about the `.mocks.dart` files in this package.
```

(This also happens in the `google_maps_flutter_web` package)